src/ui/src/optionsdialog.cpp

```diff
@@ -665,10 +665,15 @@ void OptionsDialog::buildShortcutsTable( bool useDefaultsOnly )
     }
 
     for ( const auto& [ action, keys ] : shortcuts ) {
+        auto shortcutActionName = ShortcutAction::actionName( action );
+        if ( shortcutActionName.isEmpty() ) {
+            continue;
```

Comment:
Avoid showing the deprecated action.

Reply:
Deprecating a shortcut key means that we need to add upgrade code that will move custom shortcuts that somebody might have configured to a new key. I don't think this case is worth it. Let's leave the original shortcut key, but change the visible name for it.